aws-proofs/vm-steps.sh

```diff
@@ -10,6 +10,8 @@
 set -e
 
 echo "::group::Setting up"
+curl https://storage.googleapis.com/git-repo-downloads/repo > ~/.local/bin
```

Comment:
What is the worst thing that could happen if the binary we download here is malicious? There is an implicit TLS certificate check here, but the binary could be anything.

Reply:
We already do the same download in several other actions, this just adds one more instance.

The worst that can happen is similar to the worst that can happen for a PR, because a malicious PR can execute arbitrary code on the test runner. In this case the test runner is on AWS, not GitHub, so it costs money and it can consume compute resources up to the timeout (the timeout is controlled externally outside the VM). It can access the internet, but it does not have any credentials to access anything that is not already public.